sourcegraph/jetbrains
# Persist repository URL in project settings (fixes #56948)

Additionally: I've removed the line stating that leaving the address empty would result in an inferred URL from the Git repository. This statement is not true: the dialog does not allow you to leave it blank.

## Test plan

- `./gradlew :runIde`
- Open project.
- (optional) Expect new value `<option name="remoteUrl" value="github.com/user/repo"/>` in `cody_project_settings.xml`.
- Open repository dialog. Change address and click OK.
- Restart IDE.
- Verify that the repository address remains the same as it was before the restart.
